connector/connect/common/src/test/resources/query-tests/explain-results/csv_from_dataset.explain

```diff
@@ -0,0 +1 @@
+LogicalRDD [c1#0, c2#0], false
```

Comment:
Oh this makes me sad. We are we using RDDs here?

Reply:
https://github.com/apache/spark/blob/39a55121888d2543a6056be65e0c74126a9d3bdf/sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala#L424-L433

https://github.com/apache/spark/blob/39a55121888d2543a6056be65e0c74126a9d3bdf/sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala#L503-L521

Reply:
On the server side, the input `csvDataset` and `jsonDataset` are still `LocalRelation`, and the above code path is converted them to `RDD`.